apollo/src/main/graphql/com/hedvig/android/owldroid/graphql/crossSells.graphql

```diff
@@ -1,7 +1,41 @@
-query CrossSells {
+query CrossSells($locale: Locale!) {
   activeContractBundles {
     potentialCrossSells {
+      title
```

Comment:
Given the large amount of duplication we have now, should we use a GraphQL Fragment? Perhaps a separate query would be appropriate for the use-case? 🤔

Reply:
Yeah I suppose a fragment is the way to go. They are so clunky to work with though..

Reply:
What is it that you feel that Fragments are clunky?

Reply:
I agree that `object.fragments.someThingFragment.field` is less nice than `object.field` 😞 The tradeoff of not having to have duplicate methods acting on the data (which we do have a few times in this PR, mind you 😄) is worth it in my experience 🙂

Reply:
Its so unfortunate that you cant reference fragments without their "parent" query, like `CrossSells.Fragments.{...}` and `Insurance.Fragments.{...}` :( I agree its still worth it though. Fixing now!